libp2p/pubsub/gossipsub.py

```diff
@@ -548,6 +649,21 @@ def _get_in_topic_gossipsub_peers_from_minus(
 
     # RPC emitters
 
+    def pack_control_msgs(
+        self,
+        ihave_msgs: List[rpc_pb2.ControlIHave],
+        graft_msgs: List[rpc_pb2.ControlGraft],
+        prune_msgs: List[rpc_pb2.ControlPrune],
+    ) -> rpc_pb2.ControlMessage:
```

Comment:
Am I understanding it correctly that this change is for piggybacking?

Reply:
I would say it's just packing the messages together and send them at once. Will open an issue for piggybacking.